_haikus/objective-cloud.md

```diff
@@ -5,5 +5,5 @@ author: Joseph Gattuso
 ---
 
 To cloud or to git<br>
-Pondered a wise rubbber duck<br>
+Pondered a wise rubber duck<br>
```

Comment:
This change was very confusing it was introduced in aabced7499197211b88fb59ad78727be4fa7d9aa, the original composition fdb8e45f1dc2275b88b677c14336384109eb0581 didn't have this error.

More oddly, the changeset that introduced it also added another file (which has since been removed and blocked by `.gitignore``).

Fwiw, `git diff --word-diff ...` and GitHub's UI are often quite helpful at reviewing changes beyond the basic `git diff`

```
commit aabced7499197211b88fb59ad78727be4fa7d9aa
Author: Joseph Gattuso <19511143+josephgattuso@users.noreply.github.com>
Date:   Wed Sep 4 22:39:04 2019 +0100

    bug: Fixing the bug in my code

diff --git a/_haikus/objective-cloud.md b/_haikus/objective-cloud.md
index 3557ffe..40b54aa 100644
--- a/_haikus/objective-cloud.md
+++ b/_haikus/objective-cloud.md
@@ -1,9 +1,9 @@
---
layout: haiku
title: [-Objective-Cloud-]{+objective-cloud+}
author: Joseph Gattuso
---

[-to-]{+To+} cloud or to [-git?-]
[-pondered the rubber ducky-]
[-such-]{+git<br>+}
{+Pondered a wise rubbber duck<br>+}
{+Such+} profound [-choices-]{+choices<br>+}
```